nnstreamer/tensor_trainer/tensor_trainer_nntrainer.cc

```diff
@@ -555,7 +555,8 @@ void NNTrainer::NNTrainerImpl::trainModel() {
   ml_logd("pid[%d], tid[%d]", pid, tid);
 
   try {
-    model->setProperty({"epochs=" + std::to_string(num_epochs)});
+    model->setProperty(
+      {"epochs=" + std::to_string(num_epochs), "save_path=" + model_save_path});
```

Comment:
Not this PR, but I wonder how we can handle it if the user wants to set the "save_best_path".